### PR TITLE
Feature to use tags details and summary with rehype

### PR DESCRIPTION
**Pull request by arobase-che**
See issue #228

https://github.com/zestedesavoir/zmarkdown/issues/228

Their is a test and a README modification.

### CODE REVIEW

**Comment by arobase-che** [comment]
Typo in the title of the commit "detail**s**"

File: packages/remark-custom-blocks/README.md [hunk 1]
```diff
@@ -64,6 +64,11 @@ unified()
       classes: 'qux-block',
       title: 'required'
     },
+    slr: {
```
Comment (vhf) [inline packages/remark-custom-blocks/README.md]: It took me a while to figure out what `slr` was standing for. Please make it more explicit, a README should provide obvious examples. 😉

Comment (arobase-che) [reply packages/remark-custom-blocks/README.md]: Ok, it totaly makes sence :+1: 

It's juste that others custom-bock's names was 3 characters long so i tried to do the same.

Comment (vhf) [reply packages/remark-custom-blocks/README.md]: The current example blocks aren't named with a semantic in mind, they are [metasyntactic variables](https://en.wikipedia.org/wiki/Metasyntactic_variable). :)

**Comment by vhf** [comment]
I'm not sure how to whitelist HTML tags to pass-through. It's outside the scope of this PR though.

**Comment by arobase-che** [comment]
I'll check, if I find something interesting, I'll do a new PR.

File: packages/remark-custom-blocks/README.md (hunk 1 above)
Comment (arobase-che) [reply packages/remark-custom-blocks/README.md]: Can i add a commit to this PR to edit that ? Or i must do another PR -_-"

Comment (vhf) [reply packages/remark-custom-blocks/README.md]: Add this change to the existing commit and change the commit message to prefix it with `remark-custom-block: ` (see other commits: https://github.com/zestedesavoir/zmarkdown/commits/master?before=ba58ce66d5e7dd185e32c642a0503d1a2d583426+70 )

**Pull request merged**